http/examples/get/client.js

```diff
@@ -0,0 +1,25 @@
+const Arrow = require('apache-arrow');
+
+const url = 'http://localhost:8000';
+
+async function getArrowData(url) {
+  try {
+    const table = await Arrow.tableFromIPC(fetch(url))
+    return table;
```

Comment:
IIUC, what the Arrow JavaScript library calls a table is not the same as what Arrow C++ calls a Table. In JavaScript, it's made up of an array of record batches.

Reply:
Record batches are just virtual. Vectors are chunked and that implies record batches.